x-pack/filebeat/input/awss3/sqs_s3_event.go

```diff
@@ -202,6 +202,21 @@ func (p *sqsS3EventProcessor) keepalive(ctx context.Context, log *logp.Logger, w
 	defer t.Stop()
 
 	for {
+		// Renew visibility.
```

Comment:
Thanks for the PR! Sorry I know it's still a draft, I just want to put a note here. Could we add a debug message here to record we changed the visibility timeout based on the config param please?

Reply:
Done